src/components/LemmaInput.vue

```diff
@@ -0,0 +1,31 @@
+<script setup lang="ts">
+import { TextInput } from '@wmde/wikit-vue-components';
+import { useMessages } from '@/plugins/MessagesPlugin/Messages';
+
+withDefaults( defineProps<{
+	modelValue: string;
+	isErrorInputTooLong: boolean;
```

Comment:
Not blocking, and probably just aesthetic, but I find this prop name to be a bit verbose (and perhaps extending this to support more errors could be a bit more cumbersome in the future). How about renaming this to `errors` and passing down an object or array to flag the errors to display?

Reply:
~~I considered that and tried to weigh the tradeoffs. T298137 only specifies this single error, Figma does have copy for a different error, "No input at all", but that should never come to pass, because the `required` attribute wouldn't even allow the form to be submitted. So adding complexity beyond the simplest thing to do for this single-purpose component could be YAGNI?~~

And I just realized that the `required` attribute can be circumvented by only typing a space, which we probably don't want to allow. Thus, we need to support at least two errors indeed. I'll update the component. Thanks for making me think about this again. :+1: